html-relative-links/projects/projects.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Home Page</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
+    <li><a href="projects/projects.html">Projects</a></li>
```

Comment:
This path is incorrect, this link wouldn't work. Make sure you test all functionality before turning in your code for review 🤓